week1/3-website/index.html

```diff
@@ -18,5 +18,178 @@
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+      <header id="header">
+        <h1>SEASON OF MIGRATION TO THE         NORTH</h1>
+        <p id="prag">"The story of a man undone by a culture that in part created him,<br> Season of Migration to the North, is a powerful and evocative<br> examination of colonization in two vastly different worlds".</p>
+        <figure>
+          <img id="img-caption" src="https://images.randomhouse.com/cover/700jpg/9781590173022" alt="Book Cover">
+          <figcaption>
+            Book Cover
+          </figcaption>
+        </figure>
+      </header>
+      <section>
+        <article id="article">
+          <h2> <u>Introduction:</u></h2>
+            <figure>
+              <img id="img-caption" src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/ae/Tayeb_saleh.jpg/250px-Tayeb_saleh.jpg" alt="Tayeb Saleh">
+              <figcaption class="caption">
```

Comment:
If all 'figcaption' elements share a style, you should be able to apply it directly to the element without issue. Redundant classes can make things messy.